h2o-py/tests/testdir_algos/stackedensemble/pyunit_stackedensemble_base_models.py

```diff
@@ -18,8 +20,8 @@
 
 
 def prepare_data(blending=False):
-    fr = h2o.import_file(path=pu.locate("smalldata/testng/higgs_train_5k.csv"))
-    target = "response"
+    fr = h2o.import_file(path=pu.locate("smalldata/junit/weather.csv"))
```

Comment:
@tomasfryda Curious why you changed the test dataset... is it smaller/quicker to run the tests?

Reply:
yes, exactly it's a smaller dataset so it should be quicker to run the tests.